exercises/flatten-array/README.md

```diff
@@ -0,0 +1,61 @@
+# Flatten Array
+
+Take a nested list and return a single flattened list with all values except nil/null.
+
+The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any nil/null values.
+
+For Example
+
+input: [1,[2,3,null,4],[null],5]
+
+output: [1,2,3,4,5]
+
+# Groovy Tips
+
+Since this exercise has difficulty 5 it doesn't come with any starter implementation.
```

Comment:
Generally, I'm against this approach since it's tedious to write boilerplate code and method signatures which are used in unit tests for each exercise, but in this case (considering explanations below) it makes sense.

We have to make sure to put this exercise in a right spot in our `config.json` to serve it to the learner as soon as possible (to train learner's ability to write Classes in Groovy)